### PR TITLE
add notes in known issue about ubuntu DNS workaround, change headers to enable hyperlink

### PR DESCRIPTION
**Pull request by kct22aws**
Update "Known Issues" section to include explanation for Ubuntu's DNS resolver workaround and why it is handled differently comparing to AL2 Cluster. Also add section headers and change bullets to different levels of headers so hyperlinks are provided.

### CODE REVIEW

**Review by jeffhataws — APPROVED** [review]

File: examples/jobs/dp-bert-launch-job.md [hunk 1]
```diff
@@ -50,12 +54,23 @@ sbatch  --exclusive --nodes=16 --wrap "srun ./run_dp_bert_large_hf_pretrain_bf16
 
 Again, the job id will be displayed by sbatch and you can follow the training by inspecting the file `slurm_<job id>.out` file generated in `~/examples/dp_bert_hf_pretrain`.
 
+### Cluster scalability
+
+There are two measures of scalability of a cluster: strong scaling and weak scaling. Typically, for model training, the need is to speed up training run, because usage cost is determined by sample throughput for rounds of gradient updates. In such scenario, we need to adjust gradient accumulation micro-step according to number of compute nodes. This is achieved with the following in the downloaded training shell script `run_dp_bert_large_hf_pretrain_bf16_s128.sh`:
```
Comment (awsjoshir) [inline examples/jobs/dp-bert-launch-job.md:59]: should we elaborate on the strong and weak scaling and then detail gradient accumulation ?

File: releasenotes.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 * Added a parallel cluster example that explains how to use AWS ParallelCluster to build HPC compute cluster using trn1 compute nodes to run the distributed ML training job.
 
-# Known Issues
+## Known Issues
 
-* **Name or service not known** 
+### **Name or service not known**
```
Comment (awsjoshir) [inline releasenotes.md:9]: i think with latest pcluster (3.3 and above)  this should be fixed ?

Comment (kct22aws) [reply releasenotes.md]: Fixed. See new commit: https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/17/commits/6456e8eeec7c015832c27d3f05e3d197ca642249

File: examples/jobs/dp-bert-launch-job.md (hunk 1 above)
Comment (kct22aws) [reply examples/jobs/dp-bert-launch-job.md]: Fixed. See new commit: https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/17/commits/6456e8eeec7c015832c27d3f05e3d197ca642249

**Review by rgrandhiamzn — APPROVED** [review]

**Review by 5cp — APPROVED** [review]
looks good to me

**Pull request merged**